javascript.js

```diff
@@ -239,4 +239,33 @@ seeProject.forEach((button) => button.addEventListener('click', (button) => {
       }
     }
   };
-}));
\ No newline at end of file
+}));
+// // Email validation
+// const form = document.getElementsByClassName('form-flix');
+// form.addEventListener('submit', function(event) {
+//     const email = document.getElementById('email').value;
+//     const error = document.querySelector('.error');
+//     const errorMessage = "E-mail should be lovercase";
+
+//     if (email.toLowerCase === email) {
+//         form.submit();
+//         error.textContent = errorMessage
+//     } else {
+//         even.preventDefault();
+//         error.textContent = errorMessage;
+//     }
+// })
```

Comment:
- [ ] Following the best practices, Its not advisable to leave a commented piece of code in your codebase, it makes your code base look unprofessional, I'll suggest you remove this. :smile: